src/firmware_new/proto/control.proto

```diff
@@ -1,5 +1,7 @@
 syntax = "proto2";
 
+//import "software/proto/grSim_commands.proto";
```

Comment:
why is this here?

Reply:
Whoops, there should be no changes to this file. Removed.